lib/Jsrt/ChakraCommon.h

```diff
@@ -237,6 +237,7 @@ typedef unsigned short uint16_t;
         ///     Module was not yet evaluated when JsGetModuleNamespace was called.
         /// </summary>
         JsErrorModuleNotEvaluated,
+
```

Comment:
why add this blank line?

Reply:
The next line is the start of a category, all the other categories are preceded by a blank line.

Reply:
SGTM